txjuju/_juju1.py

```diff
@@ -48,3 +49,51 @@ def _env_as_dict(self, env):
         if env.bootstrap.admin_secret:
             config["admin-secret"] = env.bootstrap.admin_secret
         return config
+
+
+class CLIHooks(object):
+
+    CFGDIR_ENVVAR = "JUJU_HOME"
+
+    def get_bootstrap_args(self, spec, to=None, cfgfile=None,
+                           verbose=False, gui=False, autoupgrade=False):
+        # Note that for Juju 1.x we ignore gui.
+        if cfgfile is not None:
+            raise ValueError("cfgfile not supported for Juju 1.x bootstrap")
```

Comment:
For 1.x, the CLI does not support a bootstrap config.

For 2.x it does (as well as the normal config files), but my initial target for this code is fake-juju which doesn't need them.  As soon as we use txjuju.cli.CLI in landscape-server we will need to do all the config file stuff for 2.x.  TBH, it would be better to use the CLI to manage the local config rather than writing to the config files directly.  However, that too isn't needed at the moment.  We can switch to doing that in the future when it's worth doing (especially since fake-juju currently doesn't support the CLI subcommands we'd need).